celavi/compute_locations.py

```diff
@@ -935,16 +935,10 @@ def capacity_projections_solar(self):
         # Some states won't have previous installations and the lat/longs will
         # turn out blank in _new_facility_locs
         _need_latlong = _new_facility_locs.loc[_new_facility_locs.lat.isna() & _new_facility_locs.long.isna()].index
-        _new_facility_locs.loc[_need_latlong, 'lat'] = states.lat.loc[
-            states.region_id_2.isin(
-                _new_facility_locs.loc[_need_latlong, 'region_id_2']
-                )
-                ].values
-        _new_facility_locs.loc[_need_latlong, 'long'] = states.long.loc[
-            states.region_id_2.isin(
-                _new_facility_locs.loc[_need_latlong, 'region_id_2']
-                )
-                ].values
+        
+        for _p in _need_latlong:
+            _new_facility_locs.loc[_p, 'lat'] = states.lat.loc[states.region_id_2 == _new_facility_locs.loc[_p, 'region_id_2']].values
+            _new_facility_locs.loc[_p, 'long'] = states.long.loc[states.region_id_2 == _new_facility_locs.loc[_p, 'region_id_2']].values
 
         _new_facility_locs['facility_type'] = 'pv power plant'
```

Comment:
This code accounts for the fact that we can have more than one manufacturing step? (Fixes: "there was only ever one edge between manufacturing and the in use node"?)

Line 943: _new_facility_locs['facility_type'] = 'pv power plant' seems like "pv power plant" is hardcoded. Maybe we could flag this?